addons/website/static/tests/tours/edit_link_popover.js

```diff
@@ -184,5 +184,26 @@ wTourUtils.registerWebsitePreviewTour('edit_link_popover', {
         extra_trigger: 'iframe html:has(.o_edit_menu_popover)',
         run: function () {}, // it's a check
     },
+    {
+        content: "Ensure that a click on the link popover link opens a new window",
+        trigger: 'iframe .o_edit_menu_popover a.o_we_url_link[target="_blank"]',
+        extra_trigger: 'iframe .o_edit_menu_popover a.o_we_full_url[target="_blank"]',
+        run: function (actions) {
+            actions.tip_widget.$anchor[0].addEventListener('click', (e) => {
```

Comment:
- `actions.tip_widget.$anchor[0]` => `this.$anchor[0]`
- `ev` instead of `e`

Why not for this step :+1: A bit tricky/hacky/potentially not perfect but better than nothing and not too bad IMO :+1: